master/internal/db/postgres_trial.go

```diff
@@ -508,13 +507,8 @@ func (db *PgDB) addTrialMetrics(
 		}
 	}
 
-	switch v := m.Metrics.AvgMetrics.Fields["epoch"].AsInterface().(type) {
-	case float64, nil:
-	default:
-		return nil, fmt.Errorf("cannot add metric with non numeric 'epoch' value got %v", v)
-	}
-
-	return rollbacks, db.withTransaction("add training metrics", func(tx *sqlx.Tx) error {
+	// TODO(hamid): deindent. this is here to reduce merge conflicts.
```

Comment:
is now a good time to deindent it?

Reply:
I'm trying to lower conflicts with https://github.com/determined-ai/determined/pull/6641